File: 1021-Banknotes-and-Coins/1021.py
banknotes = [100,50,20,10,5,2]
bankcoins = [100,50,25,10,5,1]

money = float(input())
print("NOTAS:")
for i in range(len(banknotes)):
    number = int(money) // banknotes[i]
    print("{:d} nota(s) de R$ {:.2f}".format(number, banknotes[i]))
    money = money % banknotes[i]
print("MOEDAS:")
money *= 100
for i in range(len(bankcoins)):
    number = int(money) // bankcoins[i]
    print("{:d} moeda(s) de R$ {:.2f}".format(number, bankcoins[i]/100))
    money = money % bankcoins[i]

File: 1021-Banknotes-and-Coins/1021.go
package main

import "fmt"

func main() {
	var banknotes = [6]int{100, 50, 20, 10, 5, 2}
	var bankcoins = [6]float64{100.00, 50.00, 25.00, 10.00, 5.00, 1.00}

	var money float64
	var notas int
	fmt.Scan(&money)

	fmt.Print("NOTAS:\n")
	for i := 0; i < len(banknotes); i++ {

		notas = int(money) / banknotes[i]
		fmt.Printf("%d nota(s) de R$ %d.00\n", notas, (banknotes[i]))
		if money >= float64(banknotes[i]) {
			money = money - float64(banknotes[i]*notas)
		}
	}
	money *= 100.01
	fmt.Print("MOEDAS:\n")

	for i := 0; i < len(banknotes); i++ {

		notas = int(money) / int(bankcoins[i])
		fmt.Printf("%d moeda(s) de R$ %.2f\n", notas, (bankcoins[i] / 100))
		if money >= float64(bankcoins[i]) {
			money = money - (bankcoins[i] * float64(notas))
		}
	}

}
